docs/bokeh/source/conf.py

```diff
@@ -174,6 +174,10 @@
     json_url = "https://docs.bokeh.org/switcher.json"
 else:
     json_url = "../switcher.json"
+if "dev" in version or "rc" in version:
+    version_match = f"dev-{version[:3]}"
```

Comment:
Just noting that it's not impossible that we have minor versions > 9 (see the 0.12.x string of releases) but it's also pretty unlikely with our current process.